tests/std/tests/P1206R7_vector_from_range/test.cpp

```diff
@@ -161,6 +161,11 @@ void test_vso1591034() {
     assert(counted_item::count == 6);
 }
 
+void test_LWG_3743() { // COMPILE-ONLY
+    auto r                  = ranges::subrange(views::iota(0ULL) | views::take(5), 5);
+    [[maybe_unused]] auto v = r | ranges::to<vector<unsigned long long>>(0u);

```

Comment:
Nitpick: It seems inconsistent to use uppercase `ULL` as a suffix on the preceding line, but then lowercase `u` on this line. We're globally inconsistent about what case to prefer, but immediate local consistency here would be nice.

Reply:
I would personally prefer something like `ranges::to<vector<unsigned long long>>(size_t{0})`